tests/transformers/test_transformers_model_export.py

```diff
@@ -82,6 +84,25 @@
 # - Text2TextGeneration pipeline tests
 # - Conversational pipeline tests
 
+_logger = logging.getLogger(__name__)
+
+
+def flaky(test_func):
+    def decorated(*args, **kwargs):
+        error = None
+        for i in range(3):
+            try:
+                test_func(*args, **kwargs)
+                break
+            except Exception as e:
+                error = e
+                _logger.warning(f"Attempt {i} failed")
+                time.sleep(3)
+        else:
+            raise Exception(f"Test {test_func} failed after 3 attempts: {error!r}")
+
+    return decorated
```

Comment:
/ai make the following changes to this function:

1. Use `functools.wraps` to preserve the original function signature.
2. Make the number of attempts configurable by adding an optional argument that defaults to 3.
3. Return `test_func(*args, **kwargs)`.
4. After addressing 3, I think we can remove `else` and unindet `raise Exception(...)`

Reply:
Looks fantastic, great job!

Reply:
/ai make the following changes to this function:

1. Use `functools.wraps` to preserve the original function signature.
2. Make the number of attempts configurable by adding an optional argument that defaults to 3.
3. Return `test_func(*args, **kwargs)`.
4. After addressing 3, I think we can remove `else` and unindet `raise Exception(...)`
5. Add a simple docstring.